op-supervisor/supervisor/backend/db/fromda/db.go

```diff
@@ -64,20 +65,56 @@ func (db *DB) Rewind(derivedFrom uint64) error {
 	return nil
 }
 
+// First returns the first known values, alike to Latest.
+func (db *DB) First() (derivedFrom types.BlockSeal, derived types.BlockSeal, err error) {
```

Comment:
nit: with the theme of the other words (Latest, ErrFuture), this would be better called `Earliest`.